pkg/kafka_client/client.go

```diff
@@ -13,12 +13,24 @@ const MAX_EARLIEST int64 = 100
 
 type Options struct {
 	BootstrapServers string `json:"bootstrapServers"`
+	ConsumerGroupId  string `json:"consumerGroupId"`
```

Comment:
Note: changing the configuration on the fly - I experienced it with consumer group - while a Live dashboard is referencing a Datasource will cause the Plugin to crash when the Dashboard tries to poll Kafka.  I don't think that is related to my changes and is a general plugin thing. Navigating away from the Dashboard and opening fresh clears the issue.

Reply:
Adding consumer group has no use unless the command `client.Consumer.Commit()` is performed after reading the message. I didn't put the _commit_ command there because the plugin's purpose is only for observation. Therefore, there is no need to track the consumer offsets by committing.
I don't know about the bug you mentioned, and it's better to open a new issue about it. Until then, this change is redundant.

Reply:
Ah, OK.  I'll remove.